sprints/dashboard/tests/test_utils.py

```diff
@@ -368,7 +368,7 @@ def test_get_rotations_roles_for_member():
     # When the member is on FF duty
     output = get_rotations_roles_for_member('John Doe', {'FF': ['Jake Doe', 'John Doe'], 'DD': ['Jane Doe', 'James Doe']})
     assert len(output) == 1
-    assert output[0] == 'FF-2'
+    assert output[0] == 'Future Sprint FF'
```

Comment:
Changing from `FF-N` to something else will have an implication in crafty-bot as well. Just a FYI 😇